README.md

```diff
@@ -1,53 +1,28 @@
-# gen_rpc: A scalable RPC library for Erlang-VM based languages
+# gen_rpc: a scalable RPC library for BEAM (Erlang-VM) based languages [![GitHub Actions CI][ci-img]][ci]
+
+[ci]: https://github.com/miniclip/gen_rpc
+[ci-img]: https://github.com/miniclip/gen_rpc/workflows/build/badge.svg
 
 ## Overview
 
-- Latest release: ![Tag Version](https://img.shields.io/github/tag/priestjim/gen_rpc.svg)
```

Comment:
Some stuff moved to `miniclip`.
Status per branch removed, since it's on the top, now (and only for `master`)

Reply:
👍